index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Startup Matchmaker</title>
+  <style>
+    div.content {
+      background-image: url("images/coworking.jpg");
+      background-repeat: no-repeat;
+      /*background-width: 100%;*/
+      background-size: cover;
+      /*background-position: center center;*/
+      /*background-attachment: fixed;*/
+    }
+  </style>
+  <link href="https://fonts.googleapis.com/css?family=Merriweather:300,400|Oswald:300,400,700|Source+Sans+Pro:300,400,900" rel="stylesheet">
+  <link rel="stylesheet" type="text/css" href="css/styles.css">
+</head>
+<body>
+  <header>
+    <h1>Startup Matchmaker</h1>
+    <nav>
+      <a href="#">FIND DEVELOPERS</a>
+      <a href="#">FIND DESIGNERS</a>
+      <a class ="light" href="#">How it Works</a>
+      <a class ="light" href="#">Our Team</a>
+      <a class ="light" href="#">Blog</a>
+    </nav>
+  </header>
+  <div class="tagline">
+    <h2>Because two brains <strike>taste</strike> <span>are</span> better than one.</h2>
```

Comment:
On the`<span>` if you  add a `border-bottom: 2px solid yellow` you will get the desired outcome.